
＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,038                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15392]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,051                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15405]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,051                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15406]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,052                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15406]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,779                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15133]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,053                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15407]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,053                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15408]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,054                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15408]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,054                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15409]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,055                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15409]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,055                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15410]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,056                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15410]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,056                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15410]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,056                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15411]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,057                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15411]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,057                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15412]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,058                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15412]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,058                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15413]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,059                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15413]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,059                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15414]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:43,278                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[13633]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,078                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14433]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,061                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15416]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,062                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15416]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,062                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15417]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,063                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15417]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,063                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15418]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,064                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15418]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,064                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15419]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,064                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15419]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,065                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15419]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,691                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15045]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,066                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15421]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,066                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15421]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,067                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15421]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,067                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15422]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,068                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15422]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,068                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15423]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,069                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15423]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,069                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15423]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,069                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15424]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,070                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15424]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,070                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15425]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,071                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15425]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,071                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15426]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,071                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15426]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,072                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15426]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,061                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15415]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,073                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15427]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,074                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15428]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,074                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15429]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,075                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15429]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,075                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15430]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,076                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15430]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,076                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15431]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,077                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15431]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,077                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15432]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,698                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15053]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,078                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15432]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,078                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15433]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,079                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15433]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,079                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15434]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,079                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15434]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,080                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15434]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,080                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15435]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,081                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15435]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,081                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15436]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,081                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15436]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,082                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15436]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,082                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15437]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,083                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15437]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,083                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15437]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,083                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15438]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,942                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15296]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,084                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15439]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,085                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15439]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,085                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15440]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,085                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15440]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,086                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15440]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,086                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15441]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,087                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15441]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,087                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15442]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,087                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15442]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,088                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15442]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,088                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15443]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,089                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15443]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,089                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15443]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,853                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15207]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,090                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15444]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,090                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15445]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,091                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15445]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,773                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15127]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,092                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15446]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,092                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15447]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,092                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15447]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,093                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15448]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,094                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15448]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,094                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15448]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,094                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15449]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,095                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15449]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,095                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15450]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,096                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15450]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,096                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15450]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,096                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15451]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,097                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15451]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,492                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14847]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,098                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15452]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,098                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15453]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,098                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15453]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,099                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15453]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,099                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15454]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,100                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15454]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26
